scio-examples/src/main/scala/com/spotify/scio/examples/complete/game/LeaderBoard.scala

```diff
@@ -86,6 +106,9 @@ object LeaderBoard {
                           teamWindowDuration: Duration,
                           allowedLateness: Duration): SCollection[(String, Int)] =
     infos.withFixedWindows(
+      // Using a fixed window, calculate every time the window ends, allowing recalculations
```

Comment:
`allowing recalculations` is this the comment about `ACCUMULATING_FIRED_PANES`. If it is, do you think it's clear in its current form?

Reply:
that was my attempt to comment about withEarlyFirings/withLateFirings. I read the doc on these and I'll try to rewrite it a bit clearer - if it's still not clear, I could use some help.

Reply:
See above, don't hesitate to link to the official doc of beam and/or literally copy text from their doc.